code/modules/projectiles/projectile.dm

```diff
@@ -629,10 +629,11 @@
 			return FALSE
 	else
 		var/mob/living/L = target
-		if(!direct_target)
-			var/checking = MOBILITY_USE | MOBILITY_STAND | MOBILITY_MOVE
-			if((!hit_stunned_targets && !(L.mobility_flags & checking)) || L.stat == DEAD)	// If target not able to use items, move and stand - or if they're just dead, pass over.
-				return FALSE
+		if(direct_target)
+			return TRUE
+		// If target not able to use items, move and stand - or if they're just dead, pass over.
+		if(L.stat == DEAD || (!hit_stunned_targets && HAS_TRAIT(L, TRAIT_IMMOBILIZED) && HAS_TRAIT(L, TRAIT_FLOORED) && HAS_TRAIT(L, TRAIT_HANDS_BLOCKED)))
```

Comment:
Nother semi sus line

Reply:
This one is actually documented in what it does.
I don't see any issues with it.